README.md

```diff
@@ -23,22 +23,27 @@ trees and syntax highlighting for a given Gleam file.
 
 1. [Install tree-sitter-cli]
 2. Create a `tree-sitters` directory in your home directory.
-3. Clone this repository (or symlink it) into the new `~/tree-sitters/` directory.
+3. Clone this repository (or symlink it) into the new `~/tree-sitters/`
+   directory.
 4. Run `tree-sitter parse path/to/file.gleam` to be shown the parse tree for the
    file.
 5. Run `tree-sitter highlight path/to/file.gleam` to be shown the file with
    syntax highlighting applied.
 
-[tree-sitter-gleam-rust-example]: https://github.com/J3RN/tree-sitter-gleam-rust-example
-[tree-sitter-cli]: https://github.com/tree-sitter/tree-sitter/blob/master/cli/README.md
-[Install tree-sitter-cli]: https://github.com/tree-sitter/tree-sitter/blob/master/cli/README.md#installation
+[tree-sitter-gleam-rust-example]:
+https://github.com/J3RN/tree-sitter-gleam-rust-example
+[tree-sitter-cli]:
+https://github.com/tree-sitter/tree-sitter/blob/master/cli/README.md
+[Install tree-sitter-cli]:
+https://github.com/tree-sitter/tree-sitter/blob/master/cli/README.md#installation
```

Comment:
I'm not a believer in wrapping Markdown content at a given line length.  The reason being that if you _don't_ wrap it, it can be read in a viewer/editor with line wrap at line length 60, 80, 100, etc; whatever the user wants.  If you _do_ wrap it and the user prefers a narrower line length (say, 60 characters), you get weird artifacting like this:

> 3. Clone this repository (or symlink it) into the new
`~/tree_sitters/`
directory.